### PR TITLE
Add elision token filter docs #7981

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add elision token filter docs

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/7981

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas Could you review this PR for technical accuracy?

**Comment by vagimeli** [comment]
@varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

**Comment by vagimeli** [comment]
> @varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

@varun-lodaya This is over a month old. We need tech review approval to move it forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:12]:
```suggestion
The `elision` token filter already comes preconfigured in the following [language analyzers]({{site.url}}{{site.baseurl}}/analyzers/language-analyzers/): `catalan`, `french`, `irish`, and `italian`.
```

Comment (kolchfa-aws) [reply _analyzers/token-filters/elision.md:12]: Could we link to the language analyzers (once they are merged)?

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:21]:
```suggestion
`articles` | Required if `articles_path` is not configured | Array of strings | Defines which articles or short words should be removed when they appear as part of an elision.
```

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:22]:
```suggestion
`articles_path` | Required if `articles` is not configured | String | Specifies the path to a custom list of articles that should be removed during the analysis process. 
```

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:23]:
```suggestion
`articles_case` | Optional | Boolean | Specifies whether the filter is case sensitive when matching elisions. Default is `false`.
```

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:27]: Should we mention the default set of French elisions (l', m', t', qu', n', s', j', d', c', jusqu', quoiqu', lorsqu', puisqu')?

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @AntonEliatra! Please let me know if you have any questions.

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:27]:
```suggestion
The default set of French elisions is `l'`, `m'`, `t'`, `qu'`, `n'`, `s'`, `j'`, `d'`, `c'`, `jusqu'`, `quoiqu'`, `lorsqu'`, and `puisqu'`. You can update this by configuring the `french_elision` token filter. The following example request creates a new index named `french_texts` and configures an analyzer with `french_elision` filter:
```

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [inline _analyzers/token-filters/elision.md:7]:
```suggestion 

# Elision token filter

```

File: _analyzers/token-filters/elision.md
Comment (natebower) [inline _analyzers/token-filters/elision.md:10]:
```suggestion
The `elision` token filter is used to remove elided characters from words in certain languages. Elision typically occurs in languages such as French, in which words are often contracted and combined with the following word, typically by omitting a vowel and replacing it with an apostrophe. 
```

File: _analyzers/token-filters/elision.md
Comment (natebower) [inline _analyzers/token-filters/elision.md:12]:
```suggestion
The `elision` token filter is already preconfigured in the following [language analyzers]({{site.url}}{{site.baseurl}}/analyzers/language-analyzers/): `catalan`, `french`, `irish`, and `italian`.
```

File: _analyzers/token-filters/elision.md
Comment (natebower) [inline _analyzers/token-filters/elision.md:23]: Can the filter itself be case sensitive? Or do we mean something like "Specifies whether the filter considers/applies case sensitivity when matching elisions"?

File: _analyzers/token-filters/elision.md
Comment (natebower) [inline _analyzers/token-filters/elision.md:27]:
```suggestion
The default set of French elisions is `l'`, `m'`, `t'`, `qu'`, `n'`, `s'`, `j'`, `d'`, `c'`, `jusqu'`, `quoiqu'`, `lorsqu'`, and `puisqu'`. You can update this by configuring the `french_elision` token filter. The following example request creates a new index named `french_texts` and configures an analyzer with the `french_elision` filter:
```

**Review by natebower — COMMENTED** [review]
@AntonEliatra @kolchfa-aws Please see my comment and changes and let me know if you have any questions. Thanks!

File: _analyzers/token-filters/elision.md
Comment (kolchfa-aws) [reply _analyzers/token-filters/elision.md:23]: Technically,  yes, the filter applies case sensitivity, but I think it's understood as is 😄

File: _analyzers/token-filters/elision.md
Comment (natebower) [inline _analyzers/token-filters/elision.md:27]: with "a" `french_elision` filter"?

**Pull request merged**
